trip_app/accounts/tests/test_models.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+from django.contrib.auth import get_user_model
+from django.db import IntegrityError
+
+from . import user_factory
+
+User = get_user_model()
+
+
+@pytest.mark.django_db(transaction=True)
+class TestUsers:
+    @pytest.mark.parametrize('user__is_staff', [True, False])
+    def test_create_user_staff(self, user__is_staff):
+        """ Verify the ability to create a staff/non staff user """
+        user_factory.UserFactory(is_staff=user__is_staff)
+        assert User.objects.count() == 1
+        user = User.objects.first()
+        assert user.is_staff == user__is_staff
```

Comment:
conventional: `is` instead of `==` for bool

Reply:
Updated. Thanks.